AddUserScreen.tsx

```diff
@@ -1,91 +1,207 @@
+            birthDate: this.birthdateYYYYMMDD, password: INITIAL_PASSWORD, role: this.state.userRole } }}
+        >
+          {(mutateFunction: MutationFunction<
+              {UserCreate: {id: number}},
+              {name: string, email: string, cpf: string, birthDate: string, password: string, role: UserRoleType}
+            >,
+            { loading }
+           ) => {
+            return (
+              <View style={{ alignSelf: 'center', flex: 1, justifyContent: 'center' }}>
+                <Text style={{ fontSize: 15, marginBottom: 5 }}>Nome:</Text>
+                <TextInput
+                  style={{ height: 40, width: 200, borderColor: 'gray', borderWidth: 1, marginBottom: 25 }}
+                  autoCapitalize='words'
+                  onEndEditing={(event) => this.verifyName(event.nativeEvent.text)}
+                />
+
+                <Text style={{ fontSize: 15, marginBottom: 5 }}>CPF:</Text>
+                <TextInput
+                  style={{ height: 40, width: 200, borderColor: 'gray', borderWidth: 1, marginBottom: 0 }}
+                  autoCapitalize='none'
+                  onEndEditing={(event) => this.verifyCPF(event.nativeEvent.text)}
+                />
+                <Text style={{ fontSize: 12, marginBottom: 25, fontStyle: 'italic' }}>(Utilize pontos e traços)</Text>
+
+                <Text style={{ fontSize: 15, marginBottom: 5 }}>Data de Nascimento:</Text>
+                <TextInput
+                  style={{ height: 40, width: 200, borderColor: 'gray', borderWidth: 1, marginBottom: 0 }}
+                  autoCapitalize='none'
+                  onEndEditing={(event) => this.verifyBirthdate(event.nativeEvent.text)}
+                />
+                <Text style={{ fontSize: 12, marginBottom: 25, fontStyle: 'italic' }}>(Formato: dd/mm/yyyy)</Text>
+
+                <Text style={{ fontSize: 15, marginBottom: 5 }}>E-mail:</Text>
+                <TextInput
+                  style={{ height: 40, width: 200, borderColor: 'gray', borderWidth: 1, marginBottom: 25 }}
+                  autoCapitalize='none'
+                  onEndEditing={(event) => this.verifyEmail(event.nativeEvent.text)}
+                />
+
+                <Text style={{ fontSize: 15, marginBottom: 5 }}>Função:</Text>
+                <Picker
+                  selectedValue={this.state.userRole}
+                  onValueChange={(itemValue) => {
+                    this.verifyRole(itemValue);
+                    this.setState({ userRole: itemValue });
+                  }}>
+                  <Picker.Item label="" value="empty" />
+                  <Picker.Item label="System Administrator" value="admin" />
+                  <Picker.Item label="Regular User" value="user" />
+                </Picker>
+
+                <Button
+                  title="Cadastrar"
+                  color="#9400D3"
+                  onPress={() => {
+                    this.handleButtonPress(mutateFunction);
```

Comment:
Aqui daria pra sua função `this. handleButtonPress(...)`, quando executada, retornar `() => //Restante do codigo`. Assim você colocaria direto a chamada dela em vez de `() => this.handleButtonPress(mutateFunction)`

Reply:
Aparentemente eu tive o mesmo problema que citei na pergunta do Koji referente aos arrow functions.
Não sei como passar o argumento `mutateFunction` que vem da própria mutation.

Reply:
Resolvido, passei o `mutateFunction` da mutation pra uma variável privada da classe que eu criei. Agora `mutateFunction` não é mais passada como argumento pra `handleButtonPRess`